config/sync/core.entity_form_display.paragraph.stanford_page_title_banner.default.yml

```diff
@@ -0,0 +1,39 @@
+uuid: 1c6a81c7-d849-4420-bfcf-f87a8160b5d4
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.paragraph.stanford_page_title_banner.su_image
+    - field.field.paragraph.stanford_page_title_banner.su_title_banner_header
+    - paragraphs.paragraphs_type.stanford_page_title_banner
+  module:
+    - media_library
+id: paragraph.stanford_page_title_banner.default
+targetEntityType: paragraph
+bundle: stanford_page_title_banner
+mode: default
+content:
+  status:
```

Comment:
We can hide the "published" checkbox. We don't want to allow users to hide this one.

Reply:
Updated